hal/include/HAL/Accelerometer.h

```diff
@@ -9,11 +9,14 @@
 
 #include "HAL/Types.h"
 
-enum HAL_AccelerometerRange : int32_t {
+/* clang-format off */
```

Comment:
`//` can be used for these instead of `/* */`, and should be used instead for single line comments. This goes for the other instances as well.